lib/pf/Authentication/Source/LinkedInSource.pm

```diff
@@ -27,6 +28,17 @@ has 'redirect_url' => (isa => 'Str', is => 'rw', required => 1, default => 'http
 has 'domains' => (isa => 'Str', is => 'rw', required => 1, default => 'www.linkedin.com,api.linkedin.com,static.licdn.com');
 has 'create_local_account' => (isa => 'Str', is => 'rw', default => 'no');
 
+=head2 lookup_from_provider_info
+
+Lookup the person information from the authentication hash received during the OAuth process
+
+=cut
+
+sub lookup_from_provider_info {
```

Comment:
I know, I put it this way because of pf::lookup::person which populates from LDAP for consistency

I have nothing against changing it though